docs/lending/contracts/secondary-mp.md

```diff
@@ -0,0 +1,569 @@
+<h1>Secondary Monetary Policy</h1>
+
+The `SecondaryMonetaryPolicy` contract calculates borrow rates based on the utilization in a lending market. It uses parameters derived from the target utilization and ratios at 0% and 100% utilization to define a hyperbolic dependency. The rate is dynamically adjusted based on the current utilization and the rate from the AMM (Automated Market Maker), which mints crvUSD.
+
+This design ensures that when the target utilization is met, the borrow rate in the lending market matches the borrow rate of the minting market. At 0% utilization, the rate is defined as \(\alpha \times \text{rate}_{\text{AMM}}\) and at 100% utilization as \(\beta \times \text{rate}_{\text{AMM}}\).
+
+!!!github "GitHub"
+    The source code of the `SecondaryMonetaryPolicy.vy` contract can be found on [:material-github: GitHub](https://github.com/curvefi/curve-stablecoin/blob/master/contracts/mpolicies/SecondaryMonetaryPolicy.vy).
+
+---
+
+## **Calculations**
+
+!!!colab "Google Colab Notebook"
+    An interactive Google Colab notebook that plots the interest rate depending on utilization can be found here: [https://colab.research.google.com/drive/1lU0SWtvQoJHNe7pLiKD33nYBKacljhck?usp=sharing](https://colab.research.google.com/drive/1lU0SWtvQoJHNe7pLiKD33nYBKacljhck?usp=sharing).
+
+
+### **Borrow Rate**
+
+The formula for calculating the borrow rate is as follows:
+
+$$\text{rate} = \text{rate}_{\text{AMM}} \left( r_{\text{minf}} + \frac{A}{u_{\text{inf}} - \text{utilization}} \right) + \text{shift}$$
+
+$\text{shift}$ is an additional value which shifts the entire rate curve up or down by a specified amount.[^1]
+
+[^1]: This kind of rate shift is rarely used but is applied, for example, in the wstETH lending market. The `SecondaryMonetaryPolicy` of that market does not follow the wstETH mint market but follows the wETH mint market instead, with a +4% shift applied to the rate. This is done because the "more fair" interest rate is the wETH rate plus the staking rate (which is approximately 4%).
+
+
+### **Parameters**
+
+Depending on **target utilization ( \(u_0\) )**, **rate ratio at 0% utilization ( \(\alpha\) )**, and **rate ratio at 100% utilization ( \(\beta\) )**, the coefficients for the hyperbolic dependency are calculated as follows:
+
+$$u_{\text{inf}} = \frac{(\beta - 1) \times u_0}{(\beta - 1) \times u_0 - (1 - u_0) \times (1 - \alpha)}$$
+
+$$A = (1 - \alpha) \times (u_{\text{inf}} - u_0) \times \frac{u_{\text{inf}}}{u_0}$$
```

Comment:
Could we add a note to say that this A has nothing to do with other A variables in curve?  As in it has nothing to do with the amplification coefficients?  Kinda confusing mich decided to use A again here.